ddtrace/utils/install.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+from wrapt.importer import (
+    register_post_import_hook,
+    _post_import_hooks,
```

Comment:
This is a temporary work-around until we replace wrapt with our own solution. `wrapt` doesn't provide a mechanism to remove hooks. Adding a wrapt replacement I think is outside the scope of this PR. Let me know if you this otherwise however!